docs/source/api_commands.rst

```diff
@@ -15,10 +15,37 @@ Parameters: None
 app_get_dryer_setting
 ---------------------
 
-Description:
+Description: Get docker dryer settings.
 
-Parameters:
+Parameters: None
+
+Returns:
+
+    status:
+
+    on:
+
+        cliff_on:
+
+        cliff_off
+
+        count:
+
+        dry_time: Duration dryer remains on in seconds.
 
+    off:
+    
+        cliff_on:
+
+        cliff_off:
+
+        count:
+
+Return example::
+
+    {'status': 1, 'on': {'cliff_on': 1, 'cliff_off': 1, 'count': 10, 'dry_time': 7200}, 'off': {'cliff_on': 2, 'cliff_off': 1, 'count': 10}}
```

Comment:
Put this in some md code comments

Reply:
I tried looking up how to format a code block and the only thing I could really find is using a [literal block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks).

Reply:
Ah - you're right that is the right way to do it. I'm still learning sphinx as well